src/java.base/windows/classes/java/lang/ProcessImpl.java

```diff
@@ -292,9 +302,6 @@ private static String unQuote(String str) {
         if (!str.startsWith("\"") || !str.endsWith("\"") || str.length() < 2)
```

Comment:
This method has never been doing what it says in the comment above. Maybe drop the comment altogether as the code is fairly self-explanatory?